src/python/pants/core/goals/package.py

```diff
@@ -93,6 +98,22 @@ class Package(Goal):
     subsystem_cls = PackageSubsystem
 
 
+class AllPackageableTargets(Targets):
+    pass
+
+
+@rule(desc="Find all Python targets in project", level=LogLevel.DEBUG)
```

Comment:
Should it not be "Find all Packageable targets in project" ?

Reply:
Oops! Copy+paste. Good eye!